crates/air-lib/lambda/ast/src/ast.rs

```diff
@@ -21,7 +21,7 @@ use non_empty_vec::NonEmpty;
 use serde::Deserialize;
 use serde::Serialize;
 
-// TODO: rename lambda to smth more appropriate
+// TODO: rename lambda into lens
```

Comment:
Let's not update it in this PR for better file commit history. You can do a separate PR with this line.